cli/module.schema.json

```diff
@@ -37,6 +37,21 @@
         },
         "build": {
             "$ref": "#/$defs/build_section"
+        },
+        "overrides": {
+            "type": "object",
+            "description": "Platform overrides that override other fields in meta.json. Keys here take the form 'linux/arm64', see examples. The special 'dev' key applies only in the `module reload` command.",
+            "examples": [
+                {"dev": {"entrypoint": "run.sh"}},
+                {"darwin": {"build.setup": "brew install"}},
+                {"linux/arm64": {"entrypoint": "run-arm.sh"}},
+                {"linux/any": {"entrypoint": "run-linux.sh"}}
```

Comment:
This is probably just a random example. If we do indeed want platform specific entrypoints, which makes sense if build scripts vary so much, then:

I think we'd need cloud changes to UpdateModule and the Config APIs. The executable path is constructed on the cloud side from the current entrypoint field. If we want to make it platform specific I think we'd need to amend the APIs to accept a "platform to entrypoint" mapping and store that with the registry metadata.